sdk/spring-sdk/src/test/scala/kalix/springsdk/impl/SpringSdkMessageCodecSpec.scala

```diff
@@ -67,11 +79,56 @@ class SpringSdkMessageCodecSpec extends AnyWordSpec with Matchers {
       encoded.getTypeUrl shouldBe jsonTypeUrlWith("SimpleClass")
     }
 
+    "not encode java twice" in {
+      val encoded = messageCodec.encodeJava(SimpleClass("abc", 10))
+      val reEncoded = messageCodec.encodeJava(encoded)
+      reEncoded shouldBe encoded
+    }
+
+    "not encode scala to java" in {
```

Comment:
Not so clear from the test title what we are trying to test. 

"not re-code (wrap) ScalaPbAny" ?

Reply:
Maybe "not encode when already PbAny"? 
Isn't that what we are looking for?

Reply:
yeah, wrap is a good world